products/apache-http-server.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Apache HTTP Server
+permalink: /apache
+alternate_urls:
+  - /httpd
+iconSlug: apache
+layout: post
+link: https://httpd.apache.org/
+category: server-app
+activeSupportColumn: true
+command: httpd -v
+releaseColumn: false
+releaseDateColumn: true
+sortReleasesBy: "releaseCycle"
+releases:
+  - releaseCycle: "2.4"
+    release: 2012-02-21
+    eol: false
+    latest: 2.4.52
+  - releaseCycle: "2.2"
+    release: 2005-12-01
+    eol: 2017-07-11
+---
```

Comment:
also if you want to add 2.3.x branch you can add "2.3.16" too

Reply:
2.3 was a dev branch only

Reply:
yep you are right so 2.2 and 2.4 is enough i think